packages/playground/website/cypress/e2e/blueprints.cy.ts

```diff
@@ -81,4 +81,21 @@ describe('Blueprints', () => {
 			.find('[aria-label="“Test post” (Edit)"]')
 			.should('exist');
 	});
+
+	it('PHP Shutdown should work', () => {
```

Comment:
It would be enough until the first time this file is reformatted or until the test is refactored or moved to another file. WordPress core also documents all tests with ticket numbers, I was sceptical at first but then I found it actually helpful over time.